builtin/providers/aws/resource_aws_api_gateway_integration_response.go

```diff
@@ -72,6 +72,7 @@ func resourceAwsApiGatewayIntegrationResponseCreate(d *schema.ResourceData, meta
 		RestApiId:         aws.String(d.Get("rest_api_id").(string)),
 		StatusCode:        aws.String(d.Get("status_code").(string)),
 		ResponseTemplates: aws.StringMap(templates),
+		SelectionPattern:  aws.String(d.Get("selection_pattern").(string)),
```

Comment:
@radeksimko, this is an optional parameter. What happens if we don't set anything for this? Shouldn't this be wrapped in an `if v, ok := d.GetOk(` style code block?